docs/node/activities.md

```diff
@@ -0,0 +1,68 @@
+# Activities
+
+In Temporal, Activities are typically used to represent side effects, like making an HTTP request.
```

Comment:
"side effect" is actually the name of an API used to do some "non-deterministic" logic inside the Workflow code (at least in some of the other languages). Using it here as a way to describe an Activity is not ideal I think.

```suggestion
In Temporal, Activities are typically used to interact with external resources, like making an HTTP request.
```